packages/react-scripts/config/webpack.config.js

```diff
@@ -344,10 +406,13 @@ module.exports = function(webpackEnv) {
               test: [/\.bmp$/, /\.gif$/, /\.jpe?g$/, /\.png$/],
               loader: require.resolve('url-loader'),
               options: {
-                limit: 10000,
+                limit: imageInlineSizeLimit,
+                // Swrve (START): Support media file location resolution.
                 publicPath: isEnvProduction ? '../media' : '/static/media',
                 outputPath: 'static/media',
                 name: '[name].[hash:8].[ext]',
+                // name: 'static/media/[name].[hash:8].[ext]',
```

Comment:
This name comment needed ?

Reply:
Yes, just kept it to allow reader know what was originally there. All other changes in this file are additions or commented out code, however, this specific line has actually been modified from its original form.